backend/src/utils/any.ts

```diff
@@ -1,4 +1,4 @@
-export default function any<T>(
+export function any<T>(
```

Comment:
didn't we want to remove that function? (Might be an other issue)

Reply:
Yes, thats another story. Will be sone in a follow-up